src/MakePotCommand.php

```diff
@@ -601,6 +609,20 @@ protected function extract_strings() {
 					]
 				);
 			}
+
+			if ( ! $this->skip_json ) {
+				BlockExtractor::fromDirectory(
+					$this->source,
+					$translations,
+					[
+						// Only look for block.json files, nothing else.
+						'restrictFileNames' => [ 'block.json' ],
```

Comment:
@aduth, if you still think that we should allow different names here other than `block.json, it's time to raise it now. I would be personally fine with allowing also `*.block.json` but it's it nice to have rather than something we desperately need.

Reply:
We can always change this later, or even make it configurable.

Reply:
I'm not concerned.